v2v-helper/pkg/utils/volumes_ext.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/platform9/vjailbreak/v2v-helper/vm"
+)
+
+// ToVolumeManageMap builds the request payload for manage volume.
+func ToVolumeManageMap(rdmDisk vm.RDMDisk) (map[string]interface{}, error) {
+	payload := map[string]interface{}{
+		"volume": map[string]interface{}{
+			"host": rdmDisk.CinderBackendPool,
+			"ref": map[string]string{
+				"source-name": fmt.Sprintf("volume-%s", rdmDisk.UUID),
```

Comment:
This might not always be `source-name`, we will be taking both key/value from the user in annotation/custom attribute.

Reply:
okay